### DISCUSSION TITLE
Any Alternative For Slickgrid

### DISCUSSION THREAD

**Discussion by VinayChoudhary24** [ghiscoding/Angular-Slickgrid#1176] [Q&A]
Hello, I am Using Slickgrid in my Application but I am getting thi error in console like ( It looks like you are trying to use Tree Data with Pagination but unfortunately that is simply not supported because of its complexity. ). I did read the Previous Q/A raised on this issue and seems like it was never supported and Even in the future I will be no. So If you could just guide me regarding the Slickgrid Alternatives I can Look for and Use the tree data-paginatiin..

**Comment by ghiscoding** [comment]
Simple, pay thousand of dollars for [Ag-Grid](https://www.ag-grid.com/javascript-data-grid/tree-data/) which has awesome support.
